plugins/rule_engines/irods_rule_language/src/configuration.cpp

```diff
@@ -443,13 +480,18 @@ int load_rules(const char* irbSet, const std::vector<std::string> &irbs, const i
                     getSystemFunctions( ruleEngineConfig.sysFuncDescIndex->current, ruleEngineConfig.sysRegion );
                 }
 
-                make_copy copy_rule_base_files(irbs, pid);
+                in_memory_rulebases copy_rule_base_files(irbs);
```

Comment:
These rulebases are no longer being stored in files, so consider giving this a different name.